### PR TITLE
Added metadata to existing payloads

### PR DESCRIPTION
**Pull request by sundhaug92**

### CODE REVIEW

**Comment by sundhaug92** [comment]
HFR https://forums.hak5.org/topic/42103-categorization/

File: payloads/library/dnsspoof/payload.sh [hunk 1]
```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
-# DNSSpoof payload
+# 
+# Title:		DNSSpoof
+# Description:		Forge replies to arbitrary DNS queries using DNSMasq
+# Author: 		SebKinne
```
Comment (sebkinne) [inline payloads/library/dnsspoof/payload.sh]: This should say "Hak5"

File: payloads/library/openvpn/payload.sh [hunk 2]
```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
-# OpenVPN payload
+# 
+# Title:		OpenVPN
+# Description:		Create a connection to a VPN-connection to an OpenVPN-server. Optionally: Send traffic from the clients through said tunnel.
+# Author: 		SebKinne
```
Comment (sebkinne) [inline payloads/library/openvpn/payload.sh]: This should say "Hak5"

File: payloads/library/tcpdump/payload.sh [hunk 3]
```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
-# TCPDump payload v1.0
+# 
+# Title:		TCPDump
+# Description:		Dumps networking-data to USB storage. Completes on button-press or storage full.
+# Author: 		SebKinne
```
Comment (sebkinne) [inline payloads/library/tcpdump/payload.sh]: This should say "Hak5"

File: payloads/switch1/payload.sh [hunk 4]
```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+# 
+# Title:	    default
```
Comment (sebkinne) [inline payloads/switch1/payload.sh]: This should be "Default Payload"

File: payloads/switch1/payload.sh [hunk 5]
```diff
@@ -1,4 +1,12 @@
...
+# Description:  Sets the NETMODE to NAT, then sets the LED to ATTACK
+# Author: 	    SebKinne
```
Comment (sebkinne) [inline payloads/switch1/payload.sh]: This should be "Hak5"

**Pull request merged**